webpack.common.config.js

```diff
@@ -132,10 +247,24 @@ module.exports = {
                 use: 'babel-loader'
             },
             {
-                test: /\.coffee$/,
-                exclude: /node_modules/,
-                use: 'coffee-loader'
+                test: path.resolve(__dirname, 'common/static/coffee/src/ajax_prefix.js'),
+                use: [
+                    'babel-loader',
+                    {
+                        loader: 'exports-loader',
+                        options: {
+                            'this.AjaxPrefix': true
+                        }
+                    }
+                ]
             },
+            // Until everything is packaged with webpack, including both the coffee-loader and the babel-loader
+            // is confusing, because the compiled coffeescript files are being picked up by the babel-loader
+            // {
+            //     test: /\.coffee$/,
+            //     exclude: /node_modules/,
+            //     use: 'coffee-loader'
+            // },
```

Comment:
As above, I feel like this is temporary, and I didn't want to forget to re-enable the coffee loader (because I don't want to be running coffeescript separately). Although, maybe we can just put the coffee loader back in at the point that we turn of the external coffeescript (if we do that before we just delete all the coffeescript).